schemas/io-package.json

```diff
@@ -1130,6 +1130,17 @@
             "singleton": {
               "description": "If true, only one TAB for all instances will be shown.",
               "type": "boolean"
+            },
+            "sendTo": {
```

Comment:
I don't get what this is doing by the description

Reply:
Any why boolean or string this isn't the best design if this is newly introduced.

Reply:
How would you do it? This attribute describes if the message must be sent when the GUI initializing:
- true - send 'tab' message
- anyString - send 'anyString' message to backend, so it can create an answer for JSON Config

Reply:
Still don’t get what it is doing, why isn’t it a json config flag but an io package flag? Why not set string to „tab“ to send „tab“?